inst/templates/Makefile

```diff
@@ -0,0 +1,29 @@
+R = R --vanilla CMD
+
+
+all: build check install clean
+
+build:
+	$(R) build .
+
+check: build
+	$(R) check --no-manual *.tar.gz
```

Comment:
Rather than relying on local directory and `*.tar.gz` (which could match non-package tarballs) maybe take a page from the knitr makefiles

https://github.com/yihui/knitr/blob/dc5ead7bcfc0ebd2789fe99c527c7d91afb3de4a/Makefile#L1-L4